alibi/datasets/default.py

```diff
@@ -26,7 +25,7 @@
                        'http://www.cs.cornell.edu/People/pabo/movie-review-data/rt-polaritydata.tar.gz']
 
 #  TODO change storage format.
-IMAGENET_URLS = ['https://storage.googleapis.com/seldon-datasets/imagenet10/imagenet10.joblib']
+IMAGENET_URLS = ['https://storage.googleapis.com/seldon-datasets/imagenet10/imagenet10.tar.gz']
 
 
 def fetch_imagenet_10(url_id: int = 0) -> Dict:
```

Comment:
Please add a docstring for `url_id`, i.e. `Index specifying which URL to use for downloading`